### PR TITLE
Fixes #357 — Document "bind error: address in use"

### LINKED ISSUE #357: [documentation] Add "bind: address already in use" solution to Troubleshooting section
Again, on behalf of @anyangocynthia:

Could we add the error "Error response from daemon: Cannot start container <container_id>: failed to create endpoint <name> on network bridge: Error starting userland proxy: listen tcp 0.0.0.0:5432: bind: address already in use '' to the troubleshooting section and perhaps add a solution?
 
A lot of users have encountered that problem when using docker for Kinto.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -82,3 +82,22 @@ can also happen with other database encoding. The encoding expected by kinto is
...
+
+bind: address already in use
+============================
+
+You will probably have a more precise error message telling you which
+port is already in use: ``listen tcp 0.0.0.0:5432: bind: address
+already in use``.
+
+This happens when you are trying to start a docker image on the same
+port of an existing service running on your machine.
```
Comment (leplatrem) [inline docs/troubleshooting.rst]: Or if you start kinto in two different terminals

File: docs/troubleshooting.rst [hunk 2]
```diff
@@ -82,3 +82,22 @@ can also happen with other database encoding. The encoding expected by kinto is
...
+
+You can either stop the local service::
```
Comment (leplatrem) [inline docs/troubleshooting.rst]: For example, with postgresql, ...

File: docs/troubleshooting.rst (hunk 1 above)
Comment (Natim) [inline docs/troubleshooting.rst]: This is already describe earlier: https://github.com/Kinto/kinto/pull/637/files/cd2788b4649f3a520c1c0e105e640442c174ac5c#diff-4f31c1e2c9e43e4d6edbcd36ed58f8b8R38

**Comment by leplatrem** [comment]
r+

**Pull request merged**